portality/static/js/edges/publisher.journals.edge.js

```diff
@@ -165,7 +171,7 @@ $.extend(true, doaj, {
                     category: "top-pager",
                     renderer : doaj.renderers.newPagerRenderer({
                         numberFormat: countFormat,
-                        scrollSelector: "#top-pager"
+                        scroll: false
```

Comment:
We could in theory, yes.  Let me think about it for a bit.  Do we remove code that we're not currently using, that exists and would work if we did need it, or is it ok to let it lie dormant in case we need it later?  I'm not sure of the answer.

Reply:
The question also is whether it would be used at all in the future or maybe from now the new approach will be used everywhere.

Reply:
Ok, I agree, we should strip this functionality out of our custom pager, as it is not useful, and we can always just bring it back later if we need it.

One caveat: this scroll mechanism was actually in use in the fixed query widget, but there was a note to say that it didn't actually function.  I took the decision, therefore, to remove it from the fixed query widget too.

@Steven-Eardley This means that this patch now required a rebuild of the fixed query widget, which I have not done yet, to avoid complicating the PR, but will need to be done before release.